pom.xml

```diff
@@ -11,23 +11,22 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.6</version>
+        <version>3.42</version>
     </parent>
 
     <artifactId>azure-app-service</artifactId>
     <version>0.4.3-SNAPSHOT</version>
```

Comment:
Should we add the version as `0.5.0` to indicates some breaking change for user uses old jenkins version.

*I am not sure about this, please check how a software handle the version with this situation*

Reply:
Yes, I will release the next version as 1.0.0 .